nginx/kitspace.template

```diff
@@ -68,20 +68,14 @@ server {
 		add_header                 'Access-Control-Allow-Origin'      '@KITSPACE_URL' always;
 		add_header                 'Access-Control-Allow-Methods'     'GET, POST, PUT, PATCH, DELETE, OPTIONS';
 		add_header                 'Access-Control-Allow-Credentials' 'true' always;
-		add_header                 'Access-Control-Allow-Headers'     'cache-control, pragma, user-agent';
+		add_header                 'Access-Control-Allow-Headers'     'cache-control, pragma, user-agent, content-type';
 		proxy_cookie_path          ~^(.+)$                            '$1; domain=.@KITSPACE_DOMAIN';
 		proxy_set_header           Origin                             gitea.@KITSPACE_DOMAIN;
 		proxy_pass_request_headers on;
 		client_max_body_size       @MAX_FILE_SIZE;
 
 		if ($request_method = OPTIONS) {
-			add_header Content-Length 0;
-			add_header Content-Type text/plain;
```

Comment:
We don't need these two? Content-Lenght 0 and Content-Type text/plain?

Reply:
No, `204` do the same job.